Code/GraphMol/RascalMCES/RascalMCES.cpp

```diff
@@ -117,7 +117,7 @@ void sorted_degree_seqs(
 }
 
 // Make labels for the atoms - by default the atomic symbol.
-void get_atom_labels(const ROMol &mol, const RascalOptions &opts,
+void get_atom_labels(const ROMol &mol, const RascalOptions & /* opts */,
```

Comment:
opts was passed in here in a particularly poor attempt to support chirality matching in the MCES.  When it was pointed out just how wrong it was, I took all the code out.  I would like to come back to it again at some point, but in the meantime there is no good reason for it.  If you think it would be cleaner, I could take it out for now.  It's not, and never will be, in the public API and realistically I may never get round to handle chirality.

Reply:
I fix the underscores, should I also rip out the `opt` arg, or are thinking we might add the chirality option somewhere in the future? I remove the arg before going for this fix, and it results in chain removal from some other functions too, but it's fine, I just need to bring back those changes.

Reply:
Leave the arg in, I think. It’s a long trek back to the start, and you should probably be spared having to fight through my code like that.

Reply:
Oh, it's no problem... like I said, I already did it one, and, exactly because it's a long trek, I though it might be better to leave it in place in case we wanted to implement options later on :)